src/ApiPort/project.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
   "runtimes": {
-    "win-anycpu": { }
+    "win-anycpu": { },
+    "win": { }
```

Comment:
I left the old one because Microsoft/msbuild#344 mentioned that the win- style is needed to build with RTM.